suite/src/__tests__/slow/ceramic-ceramic-integration.test.ts

```diff
@@ -94,7 +94,8 @@ const updatesAreShared = async (
   expect(StreamUtils.serializeState(doc1.state)).toEqual(StreamUtils.serializeState(doc2.state))
 }
 
-describe('Ceramic<->Ceramic multi-node integration', () => {
+// TODO: Skipped
+xdescribe('Ceramic<->Ceramic multi-node integration', () => {
```

Comment:
Why not `describe.skip`

Reply:
Yeah, could do that too. I actually like `describe.skip` better now that I think about it - the imports will remain unchanged when we re-enable the tests.